src/parser/jobs/rdm/modules/MovementSkills.js

```diff
@@ -0,0 +1,205 @@
+		this.addHook('complete', this._onComplete)
+		//So we don't get negative numbers on the final panel display
+		this._last_manafic.timestamp = this.parser.fight.start_time
+	}
+
+	_onCast(event) {
+		const abilityID = event.ability.guid
+		switch (abilityID) {
+		case ACTIONS.MANAFICATION.id:
+			//push the previous manafication instance
+			this._history.push(this._last_manafic)
+			//setup the new instance, since we're unlikely to end the fight on manafication push
+			this._last_manafic = {
+				timestamp: event.timestamp,
+				cac: 0,
+				disp: 0,
+				engagement: 0,
+				events: [],
+			}
+			break
+		case ACTIONS.DISPLACEMENT.id:
+			if (this._last_manafic) {
+				this._last_manafic.disp++
+				this._last_manafic.events.push(event)
+			}
+			break
+		case ACTIONS.ENGAGEMENT.id:
+			if (this._last_manafic) {
+				this._last_manafic.engagement++
+				this._last_manafic.events.push(event)
+			}
+			break
+		case ACTIONS.CORPS_A_CORPS.id:
+			if (this._last_manafic) {
+				this._last_manafic.cac++
+				this._last_manafic.events.push(event)
+			}
+			break
+		}
+	}
+
+	_onComplete() {
+		let cacs = 0
+		let disp = 0
+		let engagement = 0
+		let manafics = 0
+		const requirements = []
+		this._history.push(this._last_manafic)
+
+		//Parse out the final numbers
+		this._history.map(manafic => {
+			manafics++
+			cacs += manafic.cac
+			disp += manafic.disp
+			engagement += manafic.engagement
+		})
+
+		//Credit for what you did since there was no final Manafication as boss died in nearly every instance
+		//Also need to credit for there being no manafic on the opener, so you aren't dinged
+		//Over the opening cac/disp
+		//TODO: Consider coming up with Logic to detect if we came off CD right at the end or not, if we did -1 else -2
+		manafics = manafics -2
+
+		requirements.push(this._checkCac(cacs, manafics, ACTIONS.CORPS_A_CORPS.id))
+		requirements.push(...this._checkDisp(disp, engagement, ACTIONS.DISPLACEMENT.id, manafics))
+
+		//new Rule and adds the array of Requirements that just got generated
+		this.checklist.add(new Rule({
+			name: <Trans id="rdm.movementSkills.use-movement-cds">Use your Movement Skills</Trans>,
+			description: this.description,
+			requirements: requirements,
+			target: this.target,
+		}))
+	}
+
+	_checkCac(cacs, manafics, id) {
+		let threshold = manafics * CACS_PER_MANAFICATION
+		let finalValue = 0
+		threshold -= GRACE_FOR_PULL
+		if (cacs > threshold) {
+			finalValue = 100
+		} else {
+			finalValue = cacs / threshold * 100
+		}
+
+		return new Requirement({
+			name: <ActionLink {...getDataBy(ACTIONS, 'id', id)} />,
+			percent: finalValue,
+		})
+	}
+
+	_checkDisp(disps, engagements, dispID, manafics) {
+		let noFail = 0
```

Comment:
looks like this should be a boolean. given the `if` case below only sets the value, can probably just move the `if` conditional to the right side of the assignment.